ood-portal-generator/lib/ood_portal_generator/view.rb

```diff
@@ -101,6 +101,11 @@ def initialize(opts = {})
       @oidc_state_max_number_of_cookies = opts.fetch(:oidc_state_max_number_of_cookies, '10 true')
       @oidc_cookie_same_site            = opts.fetch(:oidc_cookie_same_site, @ssl ? 'Off' : 'On')
       @oidc_settings                    = opts.fetch(:oidc_settings, {})
+
+      # CORS setup
+      @api_cors_enabled                 = opts.fetch(:api_cors_enabled, false)
+      @api_cors_uri                     = opts.fetch(:api_cors_uri, '/pun/sys/dashboard/api')
+      @api_cors_origin                  = opts.fetch(:api_cors_origin, '*')
```

Comment:
We should tend towards secure defaults. I thought initially nil/empty meaning no origins but maybe the FQDN of the server (`@servername` or `@proxy_server`, somewhere in here we find the FQDN)

Reply:
If that is the preferred approach, then it needs to be nil, meaning admins will have to provide a value.

origins have to be set to the list of client domains that will be accessing the API outside the domain where the API is deployed under, so don't think that we can anticipate any values here.